core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -0,0 +1,73 @@
+package cats
+package data
+
+final case class WriterT[T[_], L, V] (run: T[(L, V)]) {
```

Comment:
Should we change `T[_]` to `F[_]`? If memory serves, everywhere else we are pretty much using `F[_]` (and `G[_]` if there are two type constructor params)